doc/man1/openssl-cmp.pod.in

```diff
@@ -0,0 +1,1142 @@
+B<-out_trusted> option), check if there is
+an IP address Subject Alternative Name matching the given IP address.
+
+=item B<-verify_email> I<email>
+
+When verification of the newly enrolled certificate is enabled (with the
+B<-out_trusted> option), check if there is
+an email address Subject Alternative Name matching the given email address.
+
+=item B<-implicit_confirm>
+
+Request implicit confirmation of newly enrolled certificates.
+
+=item B<-disable_confirm>
+
+Do not send certificate confirmation message for newly enrolled certificate
+without requesting implicit confirmation
+to cope with broken servers not supporting implicit confirmation correctly.
+B<WARNING:> This leads to behavior violating RFC 4210.
+
+=item B<-certout> I<filename>
+
+The file where the newly enrolled certificate should be saved.
+
+=back
+
+
+=head2 Certificate revocation options
+
+=over 4
+
+=item B<-oldcert> I<filename>
+
+The certificate to be updated (i.e., renewed or re-keyed) in KUR
+or to be revoked in RR.
+It must be given for RR, else it defaults to B<-cert>.
+
+The reference certificate determined in this way, if any, is also used for
+deriving default subject DN and Subject Alternative Names for IR, CR, and KUR.
+Its issuer, if any, is used as default recipient in the CMP message header
+if neither B<-srvcert>, B<-recipient>, nor B<-issuer> is available.
+
+=item B<-revreason> I<number>
+
+Set CRLReason to be included in revocation request (RR); values: C<0>..C<10>
+or C<-1> for none (which is the default).
+
+Reason numbers defined in RFC 5280 are:
+
+   CRLReason ::= ENUMERATED {
+        unspecified             (0),
+        keyCompromise           (1),
+        cACompromise            (2),
+        affiliationChanged      (3),
+        superseded              (4),
+        cessationOfOperation    (5),
+        certificateHold         (6),
+        -- value 7 is not used
+        removeFromCRL           (8),
+        privilegeWithdrawn      (9),
+        aACompromise           (10)
+    }
+
+=back
+
+
+=head2 Credentials format options
+
+=over 4
+
+=item B<-ownform> I<PEM|DER|P12>
+
+Format to try first when reading files and to use for writing files containing
+own (i.e., client-side) certificates.
+Default value is PEM.
+
+=item B<-keyform> I<PEM|DER|P12>
+
+Format to try first when reading key files.
+Default value is PEM.
+
+=item B<-otherform> I<PEM|DER|P12>
+
+Format to try first when reading files and to use for writing files containing
+certificates of other parties (i.e., the server side).
+Default value is PEM.
+
+=item B<-otherpass> I<arg>
+
+Pass phrase source for certificate given with the B<-trusted>, B<-untrusted>,
+B<-out_trusted>, B<-extracerts>, B<-tls_extra>, or B<-tls_trusted> options.
+If not given here, the password will be prompted for if needed.
+
+For more information about the format of B<arg> see the
+B<PASS PHRASE ARGUMENTS> section in L<openssl(1)>.
+
+=item B<-engine> I<id>
```

Comment:
Probably we should  also support "-provider" since providers are new for 3.0!

Reply:
Hmm, unfortunately I do not yet have any experience in using the new provider mechanism,
and I fear that adding support for it would cause major effort/delay.
Hope we can postpone this as a feature to be added whenever there is time for that.

Reply:
This change request has been fulfilled meanwhile, so resolving this.